Library/Homebrew/cmd/update-report.rb

```diff
@@ -289,6 +291,26 @@ def link_completions_manpages_and_docs(repository = HOMEBREW_REPOSITORY)
         #{e}
     EOS
   end
+
+  def migrate_gcc_dependents_if_needed
+    return if OS.mac?
+    return if Settings.read("gcc.dep.rpaths.migrated") == "true"
+
+    Formula.installed.each do |formula|
+      next unless formula.tap&.core_tap?
+      next unless formula.recursive_dependencies.map(&:name).include? "gcc"
+
+      keg = formula.installed_kegs.last
+      tab = Tab.for_keg(keg)
```

Comment:
May want to e.g. try deleting the tab and testing that this still works as expected.

Reply:
Seems to work fine. It writes a new tab with `version_scheme: -1`. The new tab is missing a lot of other information, but I imagine that's expected.